content/en/docs/configuration/acme/http01/_index.md

```diff
@@ -139,6 +139,7 @@ No other fields of the `podTemplate` exist.
 ### `ingressTemplate`
 
 It is possible to add labels and annotations to the solver ingress resources.
+It can be really useful when you are managing several Ingress Controllers across your cluster and you want to make sure that the right one will pick up and expose the solver (for the upcoming challenge to resolve).
```

Comment:
> It can be really useful when you are managing several Ingress Controllers across your cluster

Do you mean using the deprecated [ingress class annotation](https://kubernetes.io/docs/concepts/services-networking/ingress/#deprecated-annotation)? Since cert-manager v1.5 the Ingress resources are always created with `ingressClassName` field set (which the users can configure via `acme.solvers.http01.ingress.class` field on issuer spec) - as I undertand ingress classes should be the standard way how to manage multiple ingress controllers in a cluster, see [kube docs](https://kubernetes.io/docs/concepts/services-networking/ingress/#ingress-class)

Reply:
Yes you are right we can add the precision like "If you are using a old version of Ingress Controller (who rely on old annotation `kubernetes.io/ingress.class`), it can be really useful..."

What do you think ? Is it too much information ?

Reply:
> If you are using a old version of Ingress Controller (who rely on old annotation kubernetes.io/ingress.class

Those old versions of ingress controllers will anyway not work (at least the ingress-shim without `edit-in-place` annotation will not work) with cert-manager 1.5+ as since cert-manager 1.5 the temporary `Ingress` resource that _cert-manager_ creates has the `ingressClassName` field _only_. There is some discussion around what to do as this is breaking for users of some ingress implementations (see https://github.com/jetstack/cert-manager/issues/4537), but as it is now I would be a little reluctant to add that line because this PR will get merged to latest docs. What do you think?

Reply:
Yes I do agree